src/main/ionic/peers/src/assets/auth-interceptor.js

```diff
@@ -0,0 +1,18 @@
+(function (open) {
+
+  XMLHttpRequest.prototype.open = function (method, url, async, user, pass) {
```

Comment:
we should remove this part... now that real login has been added, the eventListener for status 401 should suffice